File: java/bloque1/Personaje.java
public class Personaje {
    private String nombre;
    private int nivel;
    private int hp;

    public Personaje(String nombre, int nivel, int hp) {
        this.nombre = nombre;
        this.nivel = nivel;
        this.hp = hp;
    }

    public String getNombre() { return nombre; }
    public int getNivel() { return nivel; }
    public int getHp() { return hp; }

    public void subirNivel() {
        this.nivel++;
        this.hp += 10;
    }

    public String toString() {
        return "[Nv." + nivel + "] " + nombre + " | HP:" + hp;
    }
}

File: python/bloque1/personaje.py
class Personaje:
    def __init__(self, nombre, nivel, hp):
        self.nombre = nombre
        self.nivel = nivel
        self.hp = hp

    def subir_nivel(self):
        self.nivel += 1
        self.hp += 10

    def __str__(self):
        return f'[Nv.{self.nivel}] {self.nombre} | HP:{self.hp}'
